code/game/objects/items/weapons/mop.dm

```diff
@@ -26,7 +26,7 @@ obj/item/weapon/mop/proc/clean(turf/simulated/A)
 		if(istype(F))
 			F.dirt = 0
 		for(var/obj/effect/O in A)
-			if(istype(O,/obj/effect/decal/cleanable) || istype(O,/obj/effect/overlay))
+			if(istype(O,/obj/effect/decal/cleanable))

```

Comment:
Considering this proc is also called if you click a rune, I suggest adding either a macro or an atom proc to determine if something is cleanable, and checking that instead.